cmd/cloud-gate/main.go

```diff
@@ -84,6 +86,21 @@ func main() {
 	}
 	webServer.AddHtmlWriter(logger)
 
+	//
```

Comment:
Please remove empty comment line (and blank lines)

Reply:
ack, done